Assets/GothicVR/Scripts/Caches/AssetCache.cs

```diff
@@ -69,14 +69,28 @@ public Texture2D TryGetTexture(string key)
             }
 
             var format = pxTexture.format.AsUnityTextureFormat();
-            var texture = new Texture2D((int)pxTexture.width, (int)pxTexture.height, format, (int)pxTexture.mipmapCount, false);
-            texture.name = key;
-
-            for (var i = 0u; i < pxTexture.mipmapCount; i++)
-                texture.SetPixelData(pxTexture.mipmaps[i].mipmap, (int)i);
 
-            texture.Apply();
+            Texture2D texture = null;
+            if (pxTexture.format == PxTexture.Format.tex_B8G8R8A8)
+            {
+                // Let Unity generate mips for textures with alpha, as the game doesn't provide them.
+                texture = new Texture2D((int)pxTexture.width, (int)pxTexture.height, format, true);
```

Comment:
This doesn't seem to be true. There are objects which have mipmaps even when using B8... texture format.
Wouldn't it be better to check if mipmapCount == 1 and then let Unity create them?
![image](https://github.com/GothicVRProject/GothicVR/assets/120568393/4df1c33b-be48-44fe-8f96-5474e17e1f06)


Reason for this is: If the texture is not in a Unity supported format (dxt1, dxt5) it will be converted to B8...

![image](https://github.com/GothicVRProject/GothicVR/assets/120568393/e593d605-7b5a-4e97-931c-22a676243423)

![image](https://github.com/GothicVRProject/GothicVR/assets/120568393/b8e91b36-bdc1-4c8f-8dd9-c299debd3f06)

Reply:
Honestly Unity's mips are going to be better anyway probably, but sure.

Reply:
I understand. It goes into the direction "classical G1 experience" vs improved one. I count better mipmaps to the latter. From my point we're still in the direction of the classical experience. If it won't cost performance or other awkward stuff. ;-)

Cool. After this one, we're good to go. Thank you.